packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -266,17 +304,22 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         return null;
     }
 
+    /* tslint:disable */
```

Comment:
Here again - what is it that we are disabling? If maxlines, we can simply wrap the line in question, although I would consider if the code can be refactored.